### PR TITLE
Fix apcu_key_info return value

### PR DESCRIPTION
**Pull request by kamil-tekiela**
As per https://github.com/krakjoe/apcu/blob/b68953ccbc1a5234c2caf8ed5800afd222270948/php_apc.stub.php#L12

### CODE REVIEW

File: reference/apcu/functions/apcu-key-info.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   null or array info
+   An array containing the detailed information about the cache key, or &null; on failure.
```
Comment (nikic) [inline reference/apcu/functions/apcu-key-info.xml:41]:
```suggestion
   An array containing the detailed information about the cache key, or &null; if the key does not exist.
```
Maybe?

Comment (cmb69) [reply reference/apcu/functions/apcu-key-info.xml]: Maybe both? While a non existing key is likely the cause for `null` return, `null` is also returned if locking of the cache fails: https://github.com/krakjoe/apcu/blob/b68953ccbc1a5234c2caf8ed5800afd222270948/apc_cache.c#L1145-L1147

Comment (nikic) [reply reference/apcu/functions/apcu-key-info.xml]: Locking can't really fail, it's just implemented defensively. Possibly it should just throw a fatal error instead.

Comment (cmb69) [reply reference/apcu/functions/apcu-key-info.xml]: Ah, okay, than I think we're good here. :)

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**